ios/browser/web/web_state/brave_web_state_native.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <memory>
+#include "base/memory/weak_ptr.h"
+#include "base/strings/string_piece.h"
+#include "components/sessions/core/session_id.h"
+#include "ios/web/public/web_state_observer.h"
+
+#ifndef BRAVE_IOS_BROWSER_WEB_WEB_STATE_BRAVE_WEB_STATE_NATIVE_H_
+#define BRAVE_IOS_BROWSER_WEB_WEB_STATE_BRAVE_WEB_STATE_NATIVE_H_
+
+class Browser;
+class GURL;
+class NavigationDelegate;
+
+class BraveNativeWebState final {
```

Comment:
Do we need to namespace this? Maybe instead of Brave prefix

Reply:
Fixed.